packages/run-protocol/src/vpool-xyk-amm/singlePool.js

```diff
@@ -7,89 +7,72 @@ import {
   pricesForStatedOutput,
 } from './constantProduct/calcSwapPrices.js';
 
-/**
- * @param {ZCF} zcf
- * @param {XYKPool} pool
- * @param {() => bigint} getProtocolFeeBP - retrieve governed protocol fee value
- * @param {() => bigint} getPoolFeeBP - retrieve governed pool fee value
- * @param {ZCFSeat} feeSeat
- * @param {AddLiquidityActual} addLiquidityActual
- * @returns {VPoolWrapper<SinglePoolInternalFacet>}
- */
-export const makeSinglePool = (
-  zcf,
-  pool,
-  getProtocolFeeBP,
-  getPoolFeeBP,
-  feeSeat,
-  addLiquidityActual,
-) => {
-  const secondaryBrand = pool.getSecondaryAmount().brand;
-  const centralBrand = pool.getCentralAmount().brand;
-  const getPools = () => ({
-    Central: pool.getCentralAmount(),
-    Secondary: pool.getSecondaryAmount(),
-  });
-  const publicPrices = prices => ({
-    amountIn: prices.swapperGives,
-    amountOut: prices.swapperGets,
-  });
+const getSecondaryBrand = pool => pool.getSecondaryAmount().brand;
+const getCentralBrand = pool => pool.getCentralAmount().brand;
 
-  const allocateGainsAndLosses = (seat, prices) => {
-    const poolSeat = pool.getPoolSeat();
+const getPools = pool => ({
+  Central: pool.getCentralAmount(),
+  Secondary: pool.getSecondaryAmount(),
+});
+
+export const singlePool = Far('SinglePool', {
```

Comment:
why `Far` this? It's a behavior object. I think this could be,
```suggestion
export const singlePoolBehavior = {
```

Reply:
fixed.